src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -441,6 +440,13 @@ func (r *StatefulSetResource) obj() (k8sclient.Object, error) {
 	return ss, nil
 }
 
+func overprovisioned(developerMode bool) string {
+	if developerMode {
+		return "--overprovisioned"
```

Comment:
didn't we make this an array on our live debug session?

Reply:
Yes, I just pushed new version. `rpk` will need small adjustment for the `--logger-log-level=exception=debug`. I will solve this in the next PR.

Reply:
https://github.com/vectorizedio/redpanda/pull/1156